### DISCUSSION TITLE
[Feature Request] Ability to specify date/time when an episode (or movie) was watched

### DISCUSSION THREAD

**Discussion by skst** [trakt/showly#684] [Ideas / Feature Requests]
Sometimes I forget to mark an episode (or movie) as watched. When I remember the next day (or whenever), I would like to mark it as seen on the actual day. Perhaps long-pressing the "Watched" button could bring up a date/time prompt?

Am I letting my OCD get the best of me or is anyone else interested in this?

**Comment by michaldrabik** [comment]
It will be added at some point.
